opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/activity/FamilyRegisterActivity.java

```diff
@@ -57,6 +58,7 @@ protected void registerBottomNavigation() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        Mapbox.getInstance(this, BuildConfig.MAPBOX_SDK_ACCESS_TOKEN);
         NavigationMenu.getInstance(this, null, null);
```

Comment:
We should remove the previous one from the `strings.xml` also

Reply:
removed.

Reply:
Why isn't this initialization done in the Application class?